SonataPageBundle.php

```diff
@@ -59,10 +59,18 @@ public function boot()
         }
 
         call_user_func(array($class, 'setSlugifyMethod'), function ($text) use ($container) {
+            // NEXT_MAJOR: remove this check
```

Comment:
I added some information to the `Configuration.php` in a separate commit, do you like it @Soullivaneuh